File: References/ClassSyntax.py
#Create a class to model an object
class Example:
    def _init_ (self, param1):
        self.param1 = param1

    def _str_(self):
        return f"This is a return statement when class is called in print statement."

    def myfunc(self):
        print("Code to Execute")


#Create new instance of object class
myClassVar = Example()

#Call methods of instance
myClassVar.myfunc()

#Print return string for instance
print(myClassVar)

File: References/ClassSyntax.js
//#Create a class to model an object
class Example {
    constructor (param1, param2) {
        this.param1 = param1;
        this.param2 = param2;
    }

    myMethod (attr) {
        print("Code to Execute");
    }
}

//Create new instance of object class
myClassVar = new Example()

//Call methods of instance
myClassVar.myMethod()